roles/idrac_os_deployment/defaults/main/ubuntu.yml

```diff
@@ -0,0 +1,10 @@
+ubuntu_hostname: "ubuntu"
+ubuntu_user_password: "$6$xlpCfETR.9nMepDn$0GDk0yuTOOMXrxFQacjQdbTNwCys.wMlo.EDzfKJGvLhy61R8IxaSCNveo247McGthyg7vBUgDlmTS3wF.64k1"  # Initial user password (hashed using SHA-512)
```

Comment:
Password should not be defaulted instead add a check in the [commons/validate_inputs.yml](https://github.com/Sakethanne/dellemc-openmanage-ansible-modules/blob/5fe6ce034a2c7140f28a1774595ef8154f4532bf/roles/idrac_os_deployment/tasks/common/validate_inputs.yml#L16) to ensure user has given password as a input to the role for ubuntu
Else once installed they would not know what the password was.